org/views.py

```diff
@@ -51,4 +52,41 @@ def post(self,request):
             return Response({}, status.HTTP_201_CREATED)
         else:
             data = serializer.errors
-            return Response(data, status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+            return Response(data, status.HTTP_400_BAD_REQUEST)
+
+
+@swagger_auto_schema(
+    operation_id="add_volunteer",
+    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
+    method='get',
+    responses={
+        '200': set_example({"message": "You're added as a volunteer!"}),
+        '401': set_example({"detail": "Authentication credentials were not provided."})
+    }
+)
+@api_view(['get'])
+@permission_classes([IsAuthenticated])
+def AddVolunteer(request,org_id):
+    org_count = Org.objects.filter(pk=org_id).count()
+    if org_count>0:
+        org = Org.objects.get(pk=org_id)
+        volunteer_permissions = Permissions()
+        volunteer_permission_set = PermissionSet.objects.get(
+            name='Volunteer',
+            org=org,
+            permissions = volunteer_permissions,
+        )
+        volunteer_group = Group.objects.get(
+            name='Volunteer',
+            org=org,
+            default_permission_set=volunteer_permission_set,
+        )
```

Comment:
```suggestion
        volunteer_group = Group.objects.get(
            name='Volunteer',
            org=org
        )
```

I think this line is redundant